EIPS/eip-nfr.md

```diff
@@ -0,0 +1,395 @@
+type: Standards Track
+category: ERC
+status: Draft
+created: 2022-05-08
+requires: 165, 721
+---
+
+## Simple Summary
+
+This proposal outlines the interface of a new ERC721 standard extension. NFTs can then define, process, and distribute rewards based on the realized profit to former owners.
+
+## Abstract
+
+In this Ethereum Improvement Proposal (EIP), we propose the implementation of a Future Rewards (FR) extension which will enable owners of ERC721 tokens (NFTs) to participate in future price increases after they sell their tokens.
+
+Owners of NFTs can expect to make money in two ways:
+
+1. An increase in price during their holding period;
+2. They receive Future Rewards (FRs) in the form of proceeds of the realized profits from the subsequent new owners after they have sold it. 
+
+In the event the seller is not the first owner, the original minter, the profits gained when selling an NFT will be shared with the previous owners. The same person will receive the same FR distributions under the nFR system as the next generation of new owners after them. 
+
+## Motivation
+
+Are you interested in finding something that may prove valuable in the future and acquiring it early? Excellent! In reality, you often find yourself in a predicament where it does not matter whether you are a paper hand trader or a diamond hand hodler, the price keeps rising. 
+
+Imagine if you were also rewarded with future price increases following the sale of your NFT?
+
+In addition to being desired, a feature such as this is also justified in its existence. The value of a collectible is often determined by its provenance and its ownership history. The history of ownership plays an important role in determining its value. Consequently, all parties should be compensated retrospectively for their community status, reputations, and early contributions to the price discovery process. 
+
+NFTs, in contrast to physical art and collectibles in the physical world, are not currently reflecting the contributions of their owners to their value. Since the ERC721 token can be tracked individually, and may be modified to record every change in price of any specific NFT token, there is no reason that a Future Rewards program of this type should not be established.
+
+This EIP establishes a standard interface for a profit sharing structure in all stages of the token's ownership history desired by all market participants.
+
+Additionally, as we will explain later, it discourages any "under-the-table" deals that may circumvent the rules set forth by artists and marketplaces.
+
+## Specification
+
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+We are implementing this extension using the [Open Zeppelin ERC721 set of interfaces, contracts, and utilities](https://docs.openzeppelin.com/contracts/4.x/api/token/erc721).
```

Comment:
External links not allowed.  You should be referring to EIP-721 here, not a particular implementation of it on the internet.